Dockerfile

```diff
@@ -6,7 +6,11 @@ RUN set -x \
 
 
 FROM registry.cloudogu.com/official/java:8u171-1
```

Comment:
Dogu uses a very old Java base image. Is there a reason not to switch to `java:8u242-3`?

Reply:
I think this cannot be private because otherwise it would not be converted to json correct. Final should work.

Reply:
Actually, there is a reason. The version will be changed in the password-policy-pr anyway, so there is no need to do this here. This would only result in merge conflicts.